client/blocks/product-purchase-features-list/jetpack-site-accelerator.jsx

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+
+export default localize( ( { selectedSite, translate } ) => {
```

Comment:
```
() => {
	return (
		...
	);
}
```

usually can be shortened to:

```
() => (
	...
);
```

In practice, this means:

```
export default localize( ( { selectedSite, translate } ) => {
	return (
		<div className="product-purchase-features-list__item">
			<PurchaseDetail
				icon={ <img alt="" src="/calypso/images/illustrations/jetpack-speed.svg" /> }
				title={ translate( 'Site accelerator' ) }
				description={ translate(
					'Load pages faster, and serve your images and ' +
						'static files from our global network of servers.'
				) }
				buttonText={ translate( 'View settings' ) }
				href={ '/settings/writing/' + selectedSite.slug }
			/>
		</div>
	);
} );
```

could be shortened to:

```
export default localize( ( { selectedSite, translate } ) => (
	<div className="product-purchase-features-list__item">
		<PurchaseDetail
			icon={ <img alt="" src="/calypso/images/illustrations/jetpack-speed.svg" /> }
			title={ translate( 'Site accelerator' ) }
			description={ translate(
				'Load pages faster, and serve your images and ' +
					'static files from our global network of servers.'
			) }
			buttonText={ translate( 'View settings' ) }
			href={ '/settings/writing/' + selectedSite.slug }
		/>
	</div>
) );
```

Reply:
I always forget about that structure since I'm not used to it. Will fix in a moment.

Reply:
This was totally optional FWIW - the previous version was also valid 😉 Thanks for addressing, tho!